src/analysis/locally_convex/bounded.lean

```diff
@@ -271,6 +271,30 @@ begin
   { exact λ ⟨C, hC⟩, (is_vonN_bounded_closed_ball 𝕜 E C).subset hC }
 end
 
+lemma is_vonN_bounded_iff' (s : set E) :
+  bornology.is_vonN_bounded 𝕜 s ↔ ∃ r : ℝ, ∀ (x : E) (hx : x ∈ s), ∥x∥ ≤ r :=
+begin
+  rw [normed_space.is_vonN_bounded_iff, ←metric.bounded_iff_is_bounded,
+    metric.bounded_iff_subset_ball (0 : E)],
+  split; rintro ⟨r, h⟩; use r; intros x hx,
+  { specialize h hx,
+    rwa mem_closed_ball_zero_iff at h },
+  rw mem_closed_ball_zero_iff,
+  exact h x hx,
+end
+
+lemma image_is_vonN_bounded_iff (f : E' → E) (s : set E') :
+  bornology.is_vonN_bounded 𝕜 (f '' s) ↔ ∃ r : ℝ, ∀ (x : E') (hx : x ∈ s), ∥f x∥ ≤ r :=
```

Comment:
We should probably add a ∈ version of [set.exists_image_iff](https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.exists_image_iff)

Reply:
Isn't that [set.mem_image](https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.mem_image)?

Reply:
It's [set.bex_image_iff](https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.bex_image_iff) but what we need is actually [set.ball_image_iff](https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.ball_image_iff).